third_party/build/patches/grpc/magma-grpc.diff

```diff
@@ -0,0 +1,78 @@
+diff --git a/src/core/ext/filters/client_channel/client_channel.cc b/src/core/ext/filters/client_channel/client_channel.cc
+index d2bf4f388d..26e399c2c5 100644
+--- a/src/core/ext/filters/client_channel/client_channel.cc
++++ b/src/core/ext/filters/client_channel/client_channel.cc
+@@ -457,7 +457,7 @@ get_service_config_from_resolver_result_locked(channel_data* chand) {
+         grpc_uri* uri = grpc_uri_parse(server_uri, true);
+         GPR_ASSERT(uri->path[0] != '\0');
+         service_config_parsing_state parsing_state;
+-        memset(&parsing_state, 0, sizeof(parsing_state));
++        //memset(&parsing_state, 0, sizeof(parsing_state));
```

Comment:
This does not feel like a good change. This needs a good explanation why it is necessary and why it does not introduce other bugs.